src/main/java/com/fabriccommunity/spookytime/mixin/BlockMixin.java

```diff
@@ -0,0 +1,21 @@
+package com.fabriccommunity.spookytime.mixin;
+
+import com.fabriccommunity.spookytime.registry.SpookyBlocks;
+import net.minecraft.block.Block;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+/**
+ * Makes deceased soil count as "natural" for tree spawning
+ * @author Indigo Amann
+ */
+@Mixin(Block.class)
+public class BlockMixin {
+
```

Comment:
Given I know nothing about world generation, I'd just like to point out this newline shouldn't exist.